compiler/noirc_driver/src/debug.rs

```diff
@@ -21,10 +21,14 @@ pub(crate) fn filter_relevant_files(
     let mut files_with_debug_symbols: BTreeSet<FileId> = debug_symbols
         .iter()
         .flat_map(|function_symbols| {
-            function_symbols
-                .locations
-                .values()
-                .flat_map(|call_stack| call_stack.iter().map(|location| location.file))
+            function_symbols.location_map.values().flat_map(|call_stack_id| {
+                function_symbols
+                    .location_tree
+                    .get_call_stack(*call_stack_id)
+                    .iter()
+                    .map(|location| location.file)
+                    .collect::<Vec<_>>()
```

Comment:
```suggestion
                    .into_iter()
                    .map(|location| location.file)
```
We should be able to do this which is more idiomatic as `get_call_stack` returns `Vec<Location>`

Reply:
Done